src/routes/task/task.routes.ts

```diff
@@ -0,0 +1,86 @@
+import {Router} from 'express'
+import Task from './taskModel'
+
+const router = Router()
+
+router.post('/api/tasks', async(req, res) => {
+	try {
+		const {text} = req.body
+		// Check if text includes letters or numbers
+		if( /^\s*$/.test(text)) {
+			return res.status(400).json({message: 'Text is empty!'})
+		}
+		const data = new Task({
+			text: req.body.text,
+			date: new Date().toLocaleString(),
+			description: '',
+			taskStatus: 'todo',
+			userId: req.user._id
+		})
+		const dataToSave = data.save()
+		res.status(200).json(dataToSave)
+	}
+	catch(error) {
+		res.status(500).json({message: error.message})
+	}
+})
+router.get('/api/tasks', async(req, res) => {
+	try {
+		const data = await Task.find({userId: req.user._id})
+		res.json(data)
+	}
+	catch(error){
+		res.status(500).json({message: error.message})
+	}
+})
+router.delete('/api/tasks/:id', async(req, res) => {
+	try {
+		const {id} = req.params
+		const data = await Task.findById(req.params.id)
+		if(data.userId === req.user._id) {
+			await Task.findByIdAndDelete(id)
+			res.send()

```

Comment:
https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete